scripts/create_config.js

```diff
@@ -172,11 +179,11 @@ module.exports = async callback => {
     chainConfig.bounty_registry_address = bountyRegistry.address;
     chainConfig.offer_registry_address = offerRegistry.address;
 
-    if (options && options.free) {
+    if (options && ((name == 'homechain' && options.homechain_free) || (name == 'sidechain' && options.sidechain_free))) {
```

Comment:
Do you have the accompanying changes in orchestration & infra?

Reply:
yeah gonna need to merge them all in i guess